sdk/communication/azure-communication-common/README.md

```diff
@@ -38,13 +38,13 @@ Add an `implementation` configuration to the `dependencies` block of your app's
 // build.gradle
 dependencies {
     ...
-    implementation "com.azure.android:azure-communication-common:1.0.0-beta.5"
+    implementation "com.azure.android:azure-communication-common:1.0.0-beta.7"
```

Comment:
Should this be beta.9 as JP suggested?

Reply:
Readme should contain reference to the latest "released" version. For example if we look at "Java", you can see that readme is [referring](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/communication/azure-communication-common#include-the-package) the last released version in [maven](https://search.maven.org/artifact/com.azure/azure-communication-common).

Similarly for "Android" the latest "released" version is 1.0.0-beta.7 in [maven](https://search.maven.org/artifact/com.azure.android/azure-communication-common), hence Readme use this version.

<img width="1063" alt="Screen Shot 2021-03-24 at 11 27 23 AM" src="https://user-images.githubusercontent.com/1471612/112364624-f7c69580-8c93-11eb-9e7f-b9a65cbfaae6.png">